README.md

```diff
@@ -310,12 +310,26 @@ Issue](https://help.github.com/articles/creating-an-issue/).
 
 * [Microsoft Cognitive Toolkit](https://github.com/Microsoft/CNTK)
 
+* [LightGBM](https://github.com/Microsoft/LightGBM)
+
+* [Recommenders](https://github.com/Microsoft/Recommenders)
+
+* [ML.NET](https://github.com/dotnet/machinelearning)
+
+* [Infer.NET](https://github.com/dotnet/infer)
```

Comment:
Lets remove until we have an integration story with them

Reply:
done!